components/camel-fhir/src/main/java/org/apache/camel/component/fhir/FhirJsonDataFormat.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.fhir;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+
+import ca.uhn.fhir.context.FhirContext;
+import org.apache.camel.Exchange;
+import org.apache.camel.spi.DataFormat;
+import org.apache.camel.spi.DataFormatName;
+import org.apache.camel.support.ServiceSupport;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+
+public class FhirJsonDataFormat extends ServiceSupport implements DataFormat, DataFormatName {
+
+    private FhirContext fhirContext;
+    private boolean contentTypeHeader = true;
```

Comment:
This option should also be on the model so end users can configure it there too

Reply:
Isn't it defined by default through [inheritance](https://github.com/apache/camel/blob/a030a22314eacafe416af749eb17b6a1046c4236/camel-core/src/main/java/org/apache/camel/model/DataFormatDefinition.java#L52) ?